mypy.ini

```diff
@@ -15,6 +15,9 @@ ignore_missing_imports = False
 # Only specific test files (defined in tox.ini) will be checked, so let's check everything in those.
 check_untyped_defs = True
 
+[mypy-paasta_tools.paastaapi.*]
+ignore_errors = True
```

Comment:
It seems that the generated classes don't have typing annotation?
If that's correct, can they be added somehow?  They are really useful to catch errors early on a devbox instead of doing the same later at prod.

Reply:
official openapi generator isn't targeting very modern python features. it's the same as the one used to generate kubernetes api - do we have mypy support there? otherwise we'll have to wait (or contribute) for https://github.com/triaxtec/openapi-python-client to get more mature. i didn't spend enough time to figure out what exactly isn't supported in our oapi.yaml, maybe during crustacean.

Reply:
TBH I'm not sure which kubernetes api -generated classes you're talking about.
I was looking at the [openapi-generated `EnvoyBackend` class](https://github.com/Yelp/paasta/pull/2891/files#diff-aed879f20100f44329e2acda282233d0R21) which looks like is going to replace the hand-made version from https://github.com/Yelp/paasta/blob/master/paasta_tools/envoy_tools.py#L37 .

Reply:
would it be a ok idea to wrap auto-generated class in a hand-made one and add the annotations manually? i think it's the best we can do for now. i promise to keep looking into python-native generator because it's got all the goodies - annotations, dataclasses, cleaner class hierarchy, more modern codebase, less python2-compatibility bloat, etc.

Reply:
We probably should be fine by directly using the auto-generated files for now since we don't have any alternatives anyway.
But please create a ticket about the necessity to eventually upgrade the openapi generator, so we don't forget about that.